Apps/Src/Tasks.c

```diff
@@ -60,6 +61,65 @@ OS_SEM Fault_Sem4;
 cell_asic Minions[NUM_MINIONS];
 uint32_t WDog_BitMap = 0;
 
+
+/** Pend_ResourceForCLI
+ * Pends for desired resource that CLI needs
+ * @param resourceNum the resource you want to wait for, look at CLI_Resource enum in CLI.h
+ */
+void Pend_ResourceForCLI(int resourceNum){
+    switch(resourceNum){
+        case 0:
+            OS_Sem4Pend(&SafetyCheck_Sem4);
```

Comment:
Where is this function defined? I thought the function used for pending and posting was OS_SemPend and OS_SemPost (in os.h).